src/Monolog/Logger.php

```diff
@@ -307,7 +307,7 @@ public function addRecord(int $level, string $message, array $context = []): boo
                     'level' => $level,
                     'level_name' => $levelName,
                     'channel' => $this->name,
-                    'datetime' => new DateTimeImmutable($this->microsecondTimestamps, $this->timezone),
+                    'datetime' => new \DateTimeImmutable($this->microsecondTimestamps, $this->timezone),
```

Comment:
according to :

src/Monolog/Test/TestCase.php

it should be **Monolog\DateTimeImmutable**